qcelemental/models/basis.py

```diff
@@ -52,6 +60,21 @@ def nfunctions(self) -> int:
         else:
             return sum(((L + 1) * (L + 2) // 2) for L in self.angular_momentum)
 
+    def is_contracted(self) -> bool:
+        """
+        Checks if the shell represents a contracted Gaussian or not.
+
+        Returns
+        -------
+        bool
+            True if the shell is contracted.
+        """
+
+        if (len(self.coefficients) != 1) and (len(self.angular_momentum) == 1):
```

Comment:
`return (len(self.coefficients) != 1) and (len(self.angular_momentum) == 1)`?

Reply:
Whoops :)